src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -181,9 +181,15 @@ async function writePackageJson(
 
   // Verify non-peerDependencies as they can easily lead to duplicate installs or version conflicts
   // in the node_modules folder of an application
-  const whitelist = pkg.whitelistedNonPeerDependencies.map(value => new RegExp(value));
+  const allowedList = pkg.allowedNonPeerDependencies.map(value => new RegExp(value));
   try {
-    checkNonPeerDependencies(packageJson, 'dependencies', whitelist, spinner);
+    if(pkg.whitelistedNonPeerDependencies) {
```

Comment:
This is redundant when using `x-deprecated` in the schema. This is because when using this keyword a warning is displayed when a deprecated option is used.